web/cobrands/oxfordshire/layout.scss

```diff
@@ -276,9 +272,7 @@ h4.static-with-rule {
 
 .shadow-wrap {
     ul#key-tools {
-        -webkit-box-shadow: 0em 0px 1em 1em $oxfordshire_very_light_green;
-        -moz-box-shadow: 0em 0px 1em 1em $oxfordshire_very_light_green;
-        box-shadow: 0em 0px 1em 1em $oxfordshire_very_light_green;
+        @include box-shadow(0 0 1em 1em $oxfordshire_very_light_green);
```

Comment:
Yes, due to the `make_css` improvements, FMS is pinned to libsass 3.3.6 which is on par with current sass (3.4.x).